src/bakery_manager_mobile/lib/pages/task_page.dart

```diff
@@ -24,6 +25,27 @@ class TaskPageState extends State<TaskPage> {
   void initState() {
     super.initState();
     _futureTasks = _fetchTasks();
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      final NavigatorState navigator = Navigator.of(context);
+      final MyNavigatorObserver? observer =
+          navigator.widget.observers.firstWhere(
+        (observer) => observer is MyNavigatorObserver,
+      ) as MyNavigatorObserver?;
+      if (observer != null) {
+        observer.onReturned = () async {
+          // Refetch account details when returning from another page
+          _futureTasks = _fetchTasks();
+          if(mounted) setState(() {}); // Trigger rebuild
+        };
+      }
+    });
+  }
+
+  @override
+  void didChangeDependencies() {
+    
+    super.didChangeDependencies();
+    _futureTasks = _fetchTasks();
```

Comment:
when doing _fetchTasks, i got Exception has occurred.
_Exception (Exception: Network error: type 'Null' is not a subtype of type 'String')

Reply:
when trying to create a Task

Reply:
wait I didnt pull backend yet hold on